clickhouse-jdbc/src/test/java/com/clickhouse/jdbc/JdbcIssuesTest.java

```diff
@@ -134,7 +137,8 @@ public void testIssue1373() throws SQLException {
         String sql = String.format("INSERT INTO %s (%s) SELECT %s FROM input %s", TABLE_NAME, columnNames, columnNames, columnValues);
         Connection conn = dataSource.getConnection("default", "");
         Statement st = conn.createStatement();
-        st.execute(String.format("CREATE TABLE %s (`event_id` String, `num01` Int8, `event_id_01` String) ENGINE = Log", TABLE_NAME));
+        st.execute(String.format("DROP TABLE IF EXISTS %s", TABLE_NAME));
```

Comment:
I think it will become a problem for concurrent builds because tests will use shared DB and table names for each test run are not unique. 
Adding timestamp to the table might solve the issue, but it requires extra steps for tables cleanup (if tests were aborted)

Reply:
Agree, but as you can see it still in draft

Reply:
Right. I've commented just to have a discussion about it.

Reply:
js client uses the same technic as @chernser proposed https://github.com/ClickHouse/clickhouse-js/blob/abb35e9c76377a2a688bb2e4705df13749143b71/packages/client-common/__tests__/utils/client.ts#L89

go client just creates a new instance using terraform https://github.com/ClickHouse/clickhouse-go/blob/b576bdef50952f0f6b8a26e5e6f641ca549b71f1/.github/workflows/run-tests-cloud.yml

Reply:
@Paultagoras Sergey's concern is still valid